src/aiida_quantumespresso/cli/utils/validate.py

```diff
@@ -59,12 +59,12 @@ def validate_hubbard_parameters(structure, parameters, hubbard_u=None, hubbard_v
     if hubbard_file_pk:
 
         try:
-            hubbard_file = load_node(pk=hubbard_file_pk)
-        except exceptions.NotExistent:
-            ValueError(f'{hubbard_file_pk} is not a valid pk')
+            hubbard_file = load_node(pk=hubbard_file_pk.pk)
```

Comment:
See my comment to your question. It is best to just have the function accept a node instead of a pk. Now you are accepting the node, taking its pk, and loading the corresponding node again 😅 That is a round trip for nothing